src/test/java/com/easypost/BatchTest.java

```diff
@@ -98,7 +98,7 @@ public void testAll() throws EasyPostException {
 
         assertTrue(batchesList.size() <= Fixtures.pageSize());
         assertNotNull(batches.getHasMore());
-        assertTrue(batchesList.stream().allMatch(batch -> batch instanceof Batch));
+        assertTrue(batchesList.stream().allMatch(batch -> batch != null));
```

Comment:
question: can you help me understand why we are changing these assertions? This is a much weaker assertion. The whole point of checking instance of is to ensure that we are deserializinig objects correctly.